<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>359</width>
    <height>95</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CAN загрузчик</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLineEdit" name="lineEditIP">
        <property name="text">
         <string>IP:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Группа:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QSpinBox" name="spinBoxGroup">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>32</number>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Точка:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="QSpinBox" name="spinBoxPoint">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>100</number>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="3">
       <widget class="QProgressBar" name="progressBar">
        <property name="value">
         <number>24</number>
        </property>
        <property name="textVisible">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="3" colspan="2">
       <widget class="QPushButton" name="pushButtonStart">
        <property name="text">
         <string>Загрузить</string>
        </property>
        <property name="default">
         <bool>true</bool>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="styleSheet">
    <string notr="true">QStatusBar {
    background: rgb(249, 231, 255);
}

QStatusBar::item {
    border: 1px solid red;
    border-radius: 3px;
}</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>lineEditIP</tabstop>
  <tabstop>spinBoxGroup</tabstop>
  <tabstop>spinBoxPoint</tabstop>
  <tabstop>pushButtonStart</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
